src/Dap.Protocol/Feature/Requests/ThreadsFeature.cs

```diff
@@ -2,7 +2,6 @@
 using OmniSharp.Extensions.DebugAdapter.Protocol.Models;
 using OmniSharp.Extensions.JsonRpc;
 using OmniSharp.Extensions.JsonRpc.Generation;
-using Thread = System.Threading.Thread;
```

Comment:
OMG this is all it was?! So this type came in and overrode the Thread in `Protocol.Models`...I was a bit confused by this when I brought in the update but didn't know how much of a worry it was: https://github.com/PowerShell/PowerShellEditorServices/pull/1424/files#r616249975